src/cr/cube/matrix/assembler.py

```diff
@@ -118,16 +118,13 @@ def column_weighted_bases(self):
     def columns_base(self):
         """1D/2D np.float64 ndarray of unweighted-N for each slice column/cell."""
         # --- an MR_X slice produces a 2D column-base (each cell has its own N) ---
-        rows_dim_type = self._rows_dimension.dimension_type
         columns_base = self._measures.columns_base
-        if rows_dim_type in (DT.MR_SUBVAR, DT.NUM_ARRAY):
-            return self._assemble_matrix(
-                SumSubtotals.blocks(columns_base, self._dimensions, diff_cols_nan=True)
-            )
+        if self._rows_dimension.dimension_type in (DT.MR_SUBVAR, DT.NUM_ARRAY):
```

Comment:
This remark is not related to your PR, but what would happen if you were to add `DT.CA_SUBVAR` here? How many tests would break, if any? Because I think it might be the right thing to do (although I don't immediately know what the `_assemble_matrix` would do in such case). The main point being - columns base are not really "a thing" over a set of subvariables (rows dimension).

Reply:
yes, i do agree and I think this is related to what you discovered recently on cube. 🤷🏽‍♂️